corehq/apps/app_manager/views/app_summary.py

```diff
@@ -29,6 +32,7 @@
 from corehq.apps.hqwebapp.views import BasePageView
 
 
+@method_decorator(requires_privilege_with_fallback(privileges.PROJECT_ACCESS), name='dispatch')
```

Comment:
What **do** you have access to if you don't have PROJECT_ACCESS? Would it be possible to piggyback on existing code - maybe `login_and_domain_required` - instead of needing to add this to all(?) new views?

Reply:
Having read more, I see that they also have access to billing pages so they can unpause their subscription. Still think it's worth thinking about if there's a way to invert this logic so we can decorate the exceptions instead of the rule.

Reply:
updated